apps/rehash.c

```diff
@@ -178,7 +178,7 @@ static int handle_symlink(const char *filename, const char *fullpath)
 
     for (i = 0; i < 8; i++) {
         ch = filename[i];
-        if (!isxdigit(ch))
+        if (!ossl_isxdigit(ch))
```

Comment:
Here locale-*in*dependence is more appropriate, but then again one can as well pay attention to OPENSSL_hexchar2int return value...